electrochemicalquantities.ttl

```diff
@@ -55,9 +55,16 @@ skos:prefLabel rdf:type owl:AnnotationProperty .
 :electrochemistry_0141b5c2_9f15_46f4_82e6_92a104faa476 rdf:type owl:Class ;
                                                        rdfs:subClassOf :electrochemistry_791c1915_a791_4450_acd8_7f94764743b5 ;
                                                        emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "capacity delivered during a discharge process"@en ;
+                                                       skos:altLabel "CapD"@en ;
                                                        skos:prefLabel "DischargeCapacity"@en .
 
 
+###  http://emmo.info/electrochemistry#electrochemistry_02d2d1d1_241c_429b_b4e7_31f2c3dc4835
+:electrochemistry_02d2d1d1_241c_429b_b4e7_31f2c3dc4835 rdf:type owl:Class ;
```

Comment:
We might consider to move these specific terms like names for particle sizes to an application ontology. Also including commercial formulation of electrolytes like LC30, LP47, ....

Reply:
Nevermind, I realize now that ParticleD50 is a quantity, not a specification. Id suggest moving these particle size percentiles  to `electrochemicalquantities.ttl`